integration_tests/start-pool.sh

```diff
@@ -0,0 +1,24 @@
+source integration_tests/cli/common.sh
+
+LOCALNET_DIR=".localnet"
+SED_EXT=
+
+patch_consensus_config() {
```

Comment:
I think makes sense to move that func to common.sh so both run-all and start-pool may use it